src/pages/Main/Main.js

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+import Card from 'components/Card/Card';
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
 
 const Main = () => {
-  return <h1>Abnb 메인 페이지 입니다</h1>;
+  const [itemList, setItemList] = useState([]);
+  const observerTarget = useRef(null);
+  //TODO : Back API 구현 후 추가 구현 예정
+  useEffect(() => {
+    const observer = new IntersectionObserver(entries => {
+      if (entries[0].isIntersecting) {
```

Comment:
entires[0]이 어떤걸 의미하는지 한눈에 들어오지 않습니다!
이렇게 바꿔보는건 어떨까요?
```suggestion
    const observer = new IntersectionObserver(([targetNode])=> {
      if (targetNode.isIntersecting) {
```